dash/testing/browser.py

```diff
@@ -173,6 +173,20 @@ def wait_for_element_by_css_selector(self, selector, timeout=None):
             ),
         )
 
+    def wait_for_element_by_id(self, element_id, timeout=None):
```

Comment:
Is it really worth adding this, vs `wait_for_element_by_css_selector('#mi-id')`?

Reply:
Yes the id we have like `#wait-/testing` is an invalid CSS selector

Reply:
hmm OK - seems like poor form to use a character in an ID that can't be used in CSS, but I guess sometimes we can't control that.